route_following_plugin/src/route_following_plugin.cpp

```diff
@@ -83,98 +84,97 @@ void setManeuverLaneletIds(cav_msgs::Maneuver& mvr, lanelet::Id start_id, lanele
+    config_.lane_change_plugin_= declare_parameter<std::string>("lane_change_plugin", config_.lane_change_plugin_);
+    config_.stop_and_wait_plugin_ = declare_parameter<std::string>("stop_and_wait_plugin", config_.stop_and_wait_plugin_);
+    config_.lanefollow_planning_tactical_plugin_ = declare_parameter<std::string>("lane_following_plugin", config_.lanefollow_planning_tactical_plugin_);
+    config_.route_end_point_buffer_ = declare_parameter<double>("/guidance/route/destination_downtrack_range", config_.route_end_point_buffer_);
+    config_.accel_limit_ = declare_parameter<double>("/vehicle_acceleration_limit", config_.accel_limit_);
+    config_.lateral_accel_limit_ = declare_parameter<double>("/vehicle_lateral_accel_limit", config_.lateral_accel_limit_);
+    config_.stopping_accel_limit_multiplier_ = declare_parameter<double>("stopping_accel_limit_multiplier", config_.stopping_accel_limit_multiplier_);
+    config_.min_maneuver_length_ = declare_parameter<double>("vehicle_id", config_.min_maneuver_length_);
+  }
+
+    carma_ros2_utils::CallbackReturn RouteFollowingPlugin::on_configure_plugin()
+  {
+    // Setup service servers
+    plan_maneuver_srv_= create_service<carma_planning_msgs::srv::PlanManeuvers>("plugins/" + planning_strategic_plugin_ + "/plan_maneuvers",
+                                                            std::bind(&RouteFollowingPlugin::plan_maneuvers_callback, this, std_ph::_1, std_ph::_2, std_ph::_3));
+
+    plugin_discovery_msg_.name = planning_strategic_plugin_;
+    plugin_discovery_msg_.version_id = "v1.0";
+    plugin_discovery_msg_.available = true;
+    plugin_discovery_msg_.activated = true;
+    plugin_discovery_msg_.type = carma_planning_msgs::msg::Plugin::STRATEGIC;
+    plugin_discovery_msg_.capability = "strategic_plan/plan_maneuvers";
+   
+    // Setup publishers
+    plugin_discovery_pub_ = create_publisher<carma_planning_msgs::msg::Plugin>("plugin_discovery", 1);
+    upcoming_lane_change_status_pub_ = create_publisher<carma_planning_msgs::msg::UpcomingLaneChangeStatus>("upcoming_lane_change_status", 1);
+
+    // Setup subscribers
+    twist_sub_ = create_subscription<geometry_msgs::msg::TwistStamped>("current_velocity", 50,
+                                                              std::bind(&RouteFollowingPlugin::twist_cb,this,std_ph::_1));
+    current_maneuver_plan_sub_ = create_subscription<carma_planning_msgs::msg::ManeuverPlan>("maneuver_plan", 50,
+                                                              std::bind(&RouteFollowingPlugin::current_maneuver_plan_cb,this,std_ph::_1));
+
+    // set world model point form wm listener
+   get_world_model_listener();
+
+   wm_ = get_world_model();
+
+    //set a route callback to update route and calculate maneuver
+    wml_->setRouteCallback([this]() {
+        //RCLCPP_INFO_STREAM(rclcpp::get_logger(),"Recomputing maneuvers due to a route update");
```

Comment:
re-enable all of the logs. 

`RCLCPP_INFO_STREAM(rclcpp::get_logger("route_following_plugin"), "STRING");`